src/expr/src/relation/func.rs

```diff
@@ -637,6 +638,67 @@ where
     })
 }
 
+// The expected input is in the format of [((OriginalRow, EncodedArgs), OrderByExprs...)]
```

Comment:
The tuple is structured in this way instead of `(Original Row, EncodedArgs, OrderByExprs...)` because of how `order_aggregate_datums`: it assumes the `ORDER BY` expressions start from the second field of the record.